.github/workflows/dotnet-upgrade.yml

```diff
@@ -0,0 +1,228 @@
+name: "DotNet SDK Upgrade"
+
+on:
+  schedule:
+    - cron: '0 0 * * 1'
+  workflow_dispatch:
+
+jobs:
+  dotnet-update:
+    runs-on: ubuntu-latest
+    outputs: 
+      SHOULD_UPDATE: ${{ steps.fetch_latest_patch.outputs.SHOULD_UPDATE }}
+      BRANCH_EXISTS: ${{ steps.fetch_latest_patch.outputs.BRANCH_EXISTS }}
+      DOTNET_LATEST_VERSION: ${{ steps.fetch_latest_patch.outputs.DOTNET_LATEST_VERSION }}
+      DOTNET_MINOR_VERSION: ${{ steps.fetch_current_minor_version.outputs.DOTNET_MINOR_VERSION }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Get current major minor version
+        id: fetch_current_minor_version
```

Comment:
`fetch_current_minor_version`-> `fetch_current_version`?

I hope we have the following names:
- fetch_current_version.DOTNET_MAJOR_MINOR_PATCH_VERSION
- fetch_current_version.DOTNET_MAJOR_MINOR_VERSION